charts/incubator/arksurvivalevolved/values.yaml

```
@@ -11,59 +11,59 @@ securityContext:
   readOnlyRootFilesystem: false
   runAsNonRoot: false
 
-secretEnv:
-  SRV_ADMIN_PWD: ""
-  SRV_PWD: ""
-  USERNAME: ""
-  PASSWRD: ""
+ark:
+  srv_admin_pass: ""
+  srv_password: ""
+  username: ""
+  password: ""
+  game_id: "376030"
+  rcon_enabled: true
+  game_params: "?bRawSockets"
```

```suggestion
  game_params: ""
```
https://ark.fandom.com/wiki/311.78